app/Platform/Concerns/ActsAsDeveloper.php

```diff
@@ -73,5 +74,13 @@ public function resolvedTickets(): HasMany
         return $this->hasMany(Ticket::class, 'resolver_id', 'ID');
     }
 
+    /**
+     * @return HasMany<AchievementAuthor>
+     */
+    public function achievementAuthorshipTasks(): HasMany
```

Comment:
I'm struggling with "authorship tasks". Tasks imply work to be done, and authorship implies writing. Two of the four tasks involve no writing.

Aren't these just "credits"?

Reply:
No opposition from me in changing it. Renamed to `achievementCredits()` in latest.